.github/workflows/pr-checks.yml

```diff
@@ -15,3 +15,16 @@ jobs:
         uses: "./.github/actions/bootstrap"
       - name: Validate PR Artifact
         uses: "./.github/actions/build-validate-artifact"
+  validate-quickstart-ids:
+    runs-on: ubuntu-latest
+    if: ${{ github.base_ref == 'main' && github.head_ref == 'release' }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+        with:
+          ref: ${{ github.sha }}
+      - name: Setup workspace
+        uses: "./.github/actions/bootstrap"
+      - name: Validate Quickstart IDs
+        run: |
+          cd utils && yarn validate-quickstart-ids.ts
```

Comment:
I think we missed adding the script to `package.json`. We'd need to add it and then call `cd utils && yarn validate-quickstart-ids`.

Reply:
oops, thought i did that yesterday.  fixed